### PR TITLE
set the macro PRIME_NUMBER to 251 when the complile option enable-128…

### PR DESCRIPTION
**Pull request by xiaojiqing**
set the macro PRIME_NUMBER to 251 when the compile option enable-128bit is enabled

### CODE REVIEW

**Comment by vingstar** [comment]
How about adding some  brief comments to explain the reason?

**Comment by xiaojiqing** [comment]
In SecureNN, the PrivateCompare protocol is implemented by adding all the XOR result of the two numbers to be compared and checking if the result of the accumulation of the XORs is zero. As we can see, the result tenders to become bigger when we traverse from MSB to LSB. Given in the prime equaling to 127 and the length of the integer equaling to 128 settings, the result can be zero when it reaches 127 and false comparing result will be got. That is where this pull request comes in.

File: cc/modules/protocol/mpc/comm/include/mpc_defines.h [hunk 1]
```diff
@@ -83,7 +83,13 @@ extern int NUM_OF_PARTIES;
 #define CPP_ASSEMBLY 1
 #define PARALLEL false
 #define NO_CORES 4
+
+#if ROSETTA_MPC_128
+//fix the problem of the PrivateCompare in SecureNN
```
Comment (vingstar) [inline cc/modules/protocol/mpc/comm/include/mpc_defines.h]: NOT expressive enough.

**Comment by vingstar** [comment]
LGTM. I will merge this soon after public review.

**Pull request merged**